### DISCUSSION TITLE
vc-drawing组件，绘制过程中如何获取click的Cesium.Cartesian2坐标值？

### DISCUSSION THREAD

**Discussion by 1194964459** [zouyaoji/vue-cesium#525] [Q&A] [answered]
vc-drawing组件，绘制过程中如何获取click的Cesium.Cartesian2坐标值？

在vc-viewer组件的ready方法中拿到Cesium, viewer两个参数，在需要的位置调用：
  const handler = new Cesium.ScreenSpaceEventHandler(viewer.canvas);
  const m = handler.getInputAction(Cesium.ScreenSpaceEventType.LEFT_CLICK);
  console.log("m:", m);
但 获取的方法m是undefined

**Comment by zouyaoji** [comment, marked as answer]
你好，vc-drawing绑定drawEvt可以拿到。
